component/phlare/write/write.go

```diff
@@ -60,17 +61,23 @@ func (rc *Arguments) UnmarshalRiver(f func(interface{}) error) error {
 // EndpointOptions describes an individual location for where profiles
 // should be delivered to using the Phlare push API.
 type EndpointOptions struct {
-	Name             string                   `river:"name,attr,optional"`
-	URL              string                   `river:"url,attr"`
-	RemoteTimeout    time.Duration            `river:"remote_timeout,attr,optional"`
-	Headers          map[string]string        `river:"headers,attr,optional"`
-	HTTPClientConfig *config.HTTPClientConfig `river:",squash"`
+	Name              string                   `river:"name,attr,optional"`
+	URL               string                   `river:"url,attr"`
+	RemoteTimeout     time.Duration            `river:"remote_timeout,attr,optional"`
+	Headers           map[string]string        `river:"headers,attr,optional"`
+	HTTPClientConfig  *config.HTTPClientConfig `river:",squash"`
+	MinBackoff        time.Duration            `river:"min_backoff_period,attr,optional"`  // start backoff at this level
+	MaxBackoff        time.Duration            `river:"max_backoff_period,attr,optional"`  // increase exponentially to this level
+	MaxBackoffRetries int                      `river:"max_backoff_retries,attr,optional"` // give up after this many; zero means infinite retries
```

Comment:
Should this be in a block instead of three flat attributes?  

```river
backoff {
  min_period  = ... 
  max_period  = ... 
  max_retries = ... 
}
```

(IMO the answer is yes if grouping them into a block will help guide documentation for the user).

Reply:
It's not for Loki so I went for consistency

Reply:
Ok, SGTM

Reply:
but may be we should consider it for both ?